006-pgo/attacks.tex

```diff
@@ -34,8 +34,8 @@ \chapter{Attacks\label{chap:attacks}}
   \centering
   \begin{tabular}{l r r r r|r r r r r r r}
     & \multicolumn{4}{c}{Type → Type} & \multicolumn{6}{c}{Type → Typing} & \\
-    & \multicolumn{4}{c}{\downbracefill} & \multicolumn{6}{c}{\downbracefill} & \\
-    $A$ & -2 & -1 & 0 & 1 & -3 & -2 & -1 & 0 & 1 & 2 & ARA \\
+    & \multicolumn{4}{c}{\raisebox{-4pt}{$\overbrace{\hspace{2cm}}$}} & \multicolumn{6}{c}{\raisebox{-4pt}{$\overbrace{\hspace{4cm}}$}} & \\
```

Comment:
i lost this in the rebase. let's see if it fixes the unicode-math problem... i wonder why `\downbracefill` doesn't work. i guess i could probably read `unicode-math` and figure it out.

Reply:
looks like it works perfectly! i've gone ahead and merged this plus `unicode-math` into `master`.